D1673/P1673.md

```diff
@@ -8922,10 +8922,7 @@ void matrix_product(ExecutionPolicy&& exec,
 #### Symmetric matrix-matrix product [linalg.algs.blas3.symm]
 
 <i>[Note:</i> These functions correspond to the BLAS function `xSYMM`.
-
-Unlike the symmetric rank-1 or rank-k update functions,
-these functions assume that the input matrix `A` --
-not the output matrix -- is symmetric. <i>-- end note]</i>
+<i>-- end note]</i>
```

Comment:
Thank you for deleting this Note -- the Note was not actually correct, as my comment explains below.